### PR TITLE
Chore: finish Bazel 7 migration

### PR DESCRIPTION
**Pull request by alexeagle**
Remove WORKSPACE contents. They're not needed for local dev since bzlmod is enabled, and examples/end-users don't see the WORKSPACE file.

### CODE REVIEW

File: .bazelversion [hunk 1]
```diff
@@ -0,0 +1 @@
+7.2.0rc1
```
Comment (perezd) [inline .bazelversion:1]: Is this necessary? I'd rather do 7.1.2 unless there's feature we absolutely require.

Comment (alexeagle) [reply .bazelversion:1]: It's not, but we'll have to add MODULE.bazel.lock to .gitignore since it wasn't stable

Comment (perezd) [reply .bazelversion:1]: OK, we can start there at least, if it's all the same for you? :)

**Comment by srikrsna-buf** [comment]
Can you give me access to the fork? I want to push a couple of changes

**Comment by alexeagle** [comment]
@srikrsna-buf sorry, for some reason I don't see the normal "allow maintainers to write to the PR branch" here  - I guess because this originates from an org instead of my personal? Feel free to take the commits and put them in another PR, or if you need me to re-open from my personal then I think you'd be able to push

**Pull request closed without merging**